content/documentations/_index.md

```diff
@@ -0,0 +1,26 @@
+---
+title: Documentations for Apache Camel
```

Comment:
I'd go with:
```suggestion
title: Apache Camel documentation
```

and place this file in `content/documentation`.

Reply:
I have added all the suggestions related to rephrasing of content.